### PR TITLE
Fix #1072: Use current prezto_powerline

### LINKED ISSUE #1072: Powerline prompt: Symbols broken
I have the Powerline Fonts installed, all the other Powerline themes work for me in the OS X Terminal and iTerm2.
But the "Powerline" prompt has broken symbols…

### PR DESCRIPTION
**Pull request by ericbn**
- Recently updated version of https://github.com/davidjrice/prezto_powerline 
  fixes broken symbols issue.
- Update the prezto_powerline git module to the current version commit
  (c48e4c6).

### CODE REVIEW

File: modules/prompt/external/powerline [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 8e81152340c4beb2d941340d1feb2dc29bbcc309
```
Comment (ericbn) [inline modules/prompt/external/powerline]: Why not point to `master` instead of the latest commit at that time?

**Comment by Pym** [comment]
Can someone merge this?

**Comment by kabadesu** [comment]
Nice. I'm glad I found this. I spent a couple of hours trying to figure out why I was seeing tab characters instead of the nice big triangle. I've just manually copied the newer prompt_powerline_setup for now.

**Comment by facastagnini** [comment]
This PR got merged in the community maintained fork of this repo.

zsh-users is an organization dedicated to host Zsh community projects and therefore became the rational home for a community maintained version of prezto.

Your contribution is greatly appreciated!

https://github.com/zsh-users/prezto

**Pull request closed without merging**